harness/determined/_trial_context.py

```diff
@@ -20,6 +20,7 @@ def __init__(
         self.env = env
 
         self.distributed = self._core.distributed
+        self.train = self._core.train
```

Comment:
I don't want to expose `core.train` to users, can add a getter on the TrialContext for the tensorboard path instead of handing users the whole TrainContext?

Reply:
Since all trial classes write metrics and upload profiling data for all workers already, I just removed this line.